python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -107,10 +107,20 @@ def execute(
         self._start_time = time.perf_counter()
 
         if not isinstance(dag, InputDataBuffer):
-            logger.get_logger().info("Executing DAG %s", dag)
-            logger.get_logger().info("Execution config: %s", self._options)
+            stdout_logger = logger.get_logger()
+            log_path = logger.get_datasets_log_path()
+            message = (
+                "Starting execution of Dataset. Monitor progress on Ray "
+                "Dashboard."
+            )
+            if log_path is not None:
```

Comment:
When would `log_path` be `None`? I see that it's `None` when `ray._private.worker._global_node` is `None`, but it's not obvious to me when that would happen.

Reply:
According to https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/dataset_logger.py#L65-L68, it looks like it would be `None` if using Ray client, which should not be a problem for us.

Reply:
Discussed w/ @raulchen offline, we are good to not worry about the case when `ray._private.worker._global_node` is `None`. This happens when the `ray.init` not called. For Ray Data, read APIs would trigger the `ray.init` call.